doc/rados/configuration/mclock-config-ref.rst

```diff
@@ -297,15 +297,17 @@ command can be used:
 
 After switching to the *custom* profile, the desired mClock configuration
 option may be modified. For example, to change the client reservation IOPS
-allocation for a specific OSD (say osd.0), the following command can be used:
+proportion for a specific OSD (say osd.0) to 0.5 (or 50%), the following
+command can be used:
 
   .. prompt:: bash #
 
-    ceph config set osd.0 osd_mclock_scheduler_client_res 3000
+    ceph config set osd.0 osd_mclock_scheduler_client_res 0.5
 
-.. important:: Care must be taken to change the reservations of other services like
-   recovery and background best effort accordingly to ensure that the sum of the
-   reservations do not exceed the maximum IOPS capacity of the OSD.
+.. important:: Care must be taken to change the reservations of other services
+   like recovery and background best effort accordingly to ensure that the sum
+   of the reservations do not exceed the maximum proportion (1.0) of the IOPS
```

Comment:
Does this deserve a _validator_?

Reply:
This could be a little tricky as detecting it as part of `handle_conf_change` in the mClockScheduler would be after the fact and would need additional steps to restore the earlier values. In addition, we would need a way to notify the user about this.

A validator at the config subsystem would be better and I can check if this is possible as part of a future change.